### PR TITLE
github: autolabel PRs

### PR DESCRIPTION
**Pull request by epuertat**
... starting with dashboard ones.

Signed-off-by: Ernesto Puerta <epuertat@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dashboard:
```
Comment (tchaikov) [inline .github/labeler.yml:1]: you might want to add the labeler action to our github workflow so it can read this file and label the PR.

Comment (epuertat) [reply .github/labeler.yml:1]: omg, you're right! I somehow missed adding that file 🙈

**Review by tchaikov — APPROVED** [review]

**Comment by epuertat** [comment]
Thanks a lot @tchaikov! Any clue on how/when this action becomes effective? Should we enable something in Ceph repo or just when it becomes merged?

**Comment by tchaikov** [comment]
@epuertat i use action in my own project. guess we need to get it merged before it can take effect. let's see..

**Comment by alfonsomthd** [comment]
@epuertat We should add also label with ```dashboard```:
doc/dev/developer_guide/dash-devel.rst

**Pull request merged**